Automattic/vip-go-mu-plugins
# Force QM when we should enable Search Dev Tools

## Description

On Multisite we would not enable Search Dev Tools for non Super Admins. The reason is that the `ep_get_query_log` function would not be available. 

That function is [loaded](https://github.com/Automattic/vip-go-mu-plugins/blob/develop/search/includes/classes/class-search.php#L513) if  `wpcom_vip_qm_enable` is true. 

`wpcom_vip_qm_enable` would check the same permission as we (`manage_options`) BUT only for [singlesite](https://github.com/Automattic/vip-go-mu-plugins/blob/develop/query-monitor/classes/QueryMonitor.php#L115)

This change makes it so that `ep_get_query_log` is always available.

## Changelog Description

### Plugin Updated: VIP Search

This change enables Search Dev Tools for all Administrators on multisite installations.

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test

1) Create multisite dev-env
2) Create a new subsite test
3) Create a new Admin user - vip dev-env exec --slug multisite -- wp user create --url=http://test.multisite.vipdev.lndo.site/ test2 test2@gmail.com --role='administrator' --user_pass=testtest
4) Log out and login as the new user test2
5) Disable debug mode by making [this method](https://github.com/Automattic/vip-go-mu-plugins/blob/b6eae073c07b2f4683929fd398f393e4ca6ffbf3/000-debug/debug-mode.php#L39) return false 
6) Search dev tools should still be visible
